src/backend/opencl/cpu/cpu_blas.cpp

```diff
@@ -54,9 +54,9 @@ using std::conditional;
 //                  const void *B, const int ldb, const void *beta,
 //                  void *C, const int ldc);
 #if defined(IS_OPENBLAS)
-    static const bool cplx_void_ptr = false;
+    const bool cplx_void_ptr = false;
```

Comment:
As far as I remember, static const are compile-time variables and const is a runtime variable. I don't know if the compiler can make assumptions about the value if it is just const.